docs/user-guide/kubectl-overview.md

```diff
@@ -252,6 +255,9 @@ $ kubectl describe pods/<pod-name>
 // Display the details of all the pods that are managed by the replication controller named <rc-name>.
 // Remember: Any pods that are created by the replication controller get prefixed with the name of the replication controller.
 $ kubectl describe pods <rc-name>
+
+// Describe all pods, not including uninitialized ones
+$ kubectl describe pods --include-uninitialized=false
```

Comment:
if I remember correctly, `kubectl describe pods --include-uninitialized=false` is already the default setting, right? if so, it's not worth explaining here.

Reply:
> kubectl describe pods --include-uninitialized=false is already the default setting, right?

@ahmetb uninitialized pods are included as default. So passing this flag to override the default setting.

Reply:
@dixudx is right :)

Reply:
Interesting. I recall that we decided not to include uninitialized for bulk update/list calls.

It is unintuitive that `kubectl get` does not  include uninitialized by default, but `kubectl describe` includes uninitialized by default. I'm not sure what's the reasoning behind that.